run-test.py

```diff
@@ -820,6 +820,7 @@ def deploy_postgresql(pod, service_name):
 def deploy_observer_deps():
     deploy_postgresql(OBSERVER, 'observer')
     info("starting Nginx @ pod "+str(OBSERVER))
+    ssh(OBSERVER, """sudo bash -c \\"apt update\\" """)
```

Comment:
I suggest installing postgres & nginx in base image instead of storing just cache and updating index on this step

Reply:
yes, ask @afiskon for this. but is not a subject of this changes.

Reply:
Well, if it isn't a subject, why this code present?

Reply:
this changes was made to allow this code to run in strange network environment. seems it works without it right now